Data/Sys/GameSettings/GSR.ini

```diff
@@ -0,0 +1,14 @@
+# GSRE7S - Smuggler's Run: Warzones
+[Core]
+# Values set here will override the main Dolphin settings.
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId = 4
+EmulationIssues = 
+[OnLoad]
+# Add memory patches to be loaded once on boot here.
+[OnFrame]
+[ActionReplay]
+[Video_Hacks]
+[Video_Stereoscopy]
+StereoConvergenceMinimum = 5
```

Comment:
It did not. I'm not sure why a few games didn't have ini files even though the wiki had playability states for them.